torchtune/utils/quantization.py

```diff
@@ -6,39 +6,51 @@
 
 from typing import Callable, Optional
 
-# importing TORCH_VERSION_AFTER_2_3 because `Int8DynActInt4WeightQuantizer`
-# is only available after 2.3 so we have to guard the pytorch versions to decide
-# the list of supported quantizers
-from torchao.utils import TORCH_VERSION_AFTER_2_3, TORCH_VERSION_AFTER_2_4
```

Comment:
cc @msaroufim @ebsmothers Just wanted to confirm this is OK

Reply:
Sorry to ask for the 100th time, but TORCH_VERSION_AFTER_2_4 returns True if and only if the PyTorch version is >= 2.4, right? If so we can remove both of these, since we assume everyone is on at least latest stable PyTorch

Reply:
Mark can confirm but yes I think `TORCH_VERSION_AFTER_2_4` means >= 2.4

Reply:
I believe it is true only in torchao nightlies. This was fixed. Before it was >2.4